cookbook/profiler/data_collector.rst

```diff
@@ -46,106 +46,160 @@ access to in local properties.
     store objects that cannot be serialized (like PDO objects), or you need
     to provide your own ``serialize()`` method.
 
-Most of the time, it is convenient to extend
+Most of the times, it is convenient to extend
 :class:`Symfony\\Component\\HttpKernel\\DataCollector\\DataCollector` and
 populate the ``$this->data`` property (it takes care of serializing the
 ``$this->data`` property)::
 
-    class MemoryDataCollector extends DataCollector
+    // src/AppBundle/DataCollector/MyCollector.php
+    class MyCollector extends DataCollector
     {
         public function collect(Request $request, Response $response, \Exception $exception = null)
         {
             $this->data = array(
-                'memory' => memory_get_peak_usage(true),
+                'variable' => 'value',
             );
         }
 
-        public function getMemory()
+        public function getVariable()
         {
-            return $this->data['memory'];
+            return $this->data['variable'];
         }
 
         public function getName()
         {
-            return 'memory';
+            return 'app.my_collector';
         }
     }
 
 .. _data_collector_tag:
 
-Enabling custom Data Collectors
+Enabling Custom Data Collectors
 -------------------------------
 
-To enable a data collector, add it as a regular service in one of your
-configuration, and tag it with ``data_collector``:
+To enable a data collector, define it as a regular service in your application
```

Comment:
I find "in your application" a bit too verbose, what about removing that? "To enable a data collector, define it as a regular service and tag it with ``data_collector``:"